games-util/lutris/lutris-0.5.10_beta2-r1.ebuild

```diff
@@ -63,6 +63,7 @@ PATCHES=(
 	"${FILESDIR}/${PN}-0.5.9.1-avoid-gtk-4.0-already-loaded-testerror.patch"
 	"${FILESDIR}/${P}-fix-install-requires.patch"
 	"${FILESDIR}/${P}-fix-launching-steam.patch"
+	"${FILESDIR}/choose-amdgpu-pro-vulkan-driver.patch"
```

Comment:
Please include `${PN}` and a version number in the patch. Also, can we upstream this patch? Seems like it might be useful beyond Gentoo as well.

Reply:
Is it good now ? :D

Reply:
Perfect :+1: